### ISSUE TITLE
[Bug] Builds failing on mac because Chrome is not found [1]

### ISSUE THREAD

**Issue by molant** [webhintio/hint#1772]
# 🐛 Bug report

## Description

macOS builds starting failing yesterday or the day before (01/23/2019) because [Chrome can't be found](https://dev.azure.com/webhint/webhint/_build/results?buildId=1040&view=logs&jobId=b66f3996-923d-5a48-a75d-a7c1343c48fb&taskId=549d15f7-e42b-56ae-22a7-f9a2ceea06bc&lineStart=1621&lineEnd=1621&colStart=21&colEnd=29)

```
Error (ChromeNotInstalledError) {
    code: 'ERR_LAUNCHER_NOT_INSTALLED',
    message: 'No Chrome installations found.',
}
```

We are installing Chrome in a step so... 🤔
Need to find out if it is an Azure Pipelines issue, if the installer has changed places, or what (`chrome-launcher` hasn't been updated in 4 months ([current version is 0.1.5](https://www.npmjs.com/package/chrome-launcher/v/0.10.5)).

**Comment by molant** [comment]
This seems to happen only when doing a PR and not when running master directly 😵

![image](https://user-images.githubusercontent.com/606594/51868842-d65ffe00-2303-11e9-85cf-87a0769780f8.png)

![image](https://user-images.githubusercontent.com/606594/51868934-10c99b00-2304-11e9-85d8-3e93872255e2.png)

**Comment by molant** [comment]
Also it doesn't reproduce in all PRs 😭

* Passes: https://dev.azure.com/webhint/webhint/_build/results?buildId=1117
* Fails: https://dev.azure.com/webhint/webhint/_build/results?buildId=1118

**Comment by molant** [comment]
[homebrew hasn't been updated yet](https://github.com/Homebrew/homebrew-cask/pull/58116) so that could explain todays failures. Still doesn't explain the ones before because v72 was released today.

**Comment by molant** [comment]
[It's merged](https://github.com/Homebrew/homebrew-cask/commit/59f70b04132b3d06d6faeb43f91ae232a76e6838)

**Comment by molant** [comment]
RIght now I'm manually printing the chrome version, opening and closing it in the "Install" step. So far this seems to do the trick 🤯 although I'm not 100% sure so I'll keep queueing the build to see if this is for real:

```yml
- script:
      brew cask install google-chrome
  condition: eq(variables['Agent.OS'], 'Darwin')
  displayName: 'Install Chrome'
- script: |
      /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --version
      /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome &
      sleep 10
      ps aux | grep conky | grep -v grep | awk '{print $2}' | xargs kill
  condition: eq(variables['Agent.OS'], 'Darwin')
  displayName: 'Print Chrome version'
```

**Comment by molant** [comment]
This seems to fail again so reopening 😭
In an Azure Pipelines thread they suggested installing `puppetteer` so I might try that.

**Comment by antross** [comment]
> In an Azure Pipelines thread they suggested installing `puppetteer` so I might try that.

FWIW we should already be installing `puppeteer` for some of the `extension-browser` tests...

**Comment by molant** [comment]
I've taken another look at the latest failures and the builds are timing out on macOS after 90min. Guess it's because of the extra work that needs to be done to get Chrome and such. I've increased the limit to 120min but we should find ways to reduce how long it takes to do a full test pass.

Closed via a448899804667d170fdc62e79a38a6e0ff0d00d4
